feat(xo-web): add go back to ActionButton and use it when saving a backup (#5913)
vatesfr/xen-orchestra: CHANGELOG.unreleased.md

### Enhancements

> Users must be able to say: “Nice enhancement, I'm eager to test it”

> > [backup] go back to precedent page instead of going to the overview when saving a backup job : keeps the user current filters ang page

Reviewer: ```suggestion

- [Backup] Go back to previous page instead of going to the overview when saving a backup job: keeps the user's current filters and page (PR [#5913](https://github.com/vatesfr/xen-orchestra/pull/5913))
```

### Bug fixes

> 

Reviewer: Change `xo-web patch` to `xo-web minor` in the "Packages to release" section since this is an actual feature and not only a bug fix.

> > [Backup] Go back to previous page instead of going to the overview when saving a backup job: keeps the user's current filters and page (PR [#5913](https://github.com/vatesfr/xen-orchestra/pull/5913))

Reviewer 2: ```suggestion

- [Backup] Go back to previous page instead of going to the overview when saving a backup job: keeps the user's current filters and page (PR [#5913](https://github.com/vatesfr/xen-orchestra/pull/5913))
```

> Users must be able to say: “Nice enhancement, I'm eager to test it”

> - [Backup] Go back to previous page instead of going to the overview when saving or creating a backup job: keeps the user's current filters and page (PR [#5913](https://github.com/vatesfr/xen-orchestra/pull/5913))

Reviewer 3: Job creation is no longer impacted:

```suggestion
- [Backup] Go back to previous page instead of going to the overview after editing a job: keeps current filters and page (PR [#5913](https://github.com/vatesfr/xen-orchestra/pull/5913))
```

Reviewer 3: Not fixed.
